workspace/utilities/admin-tools/widgets/widget-edit-entry.xsl

```diff
@@ -4,17 +4,27 @@
 	<xsl:template name="widget-edit-entry">
 		<xsl:param name="section" select="../section/@handle" />
 		<xsl:param name="entry-id" select="@id" />
+		<xsl:param name="domain" select="''" />
 		<xsl:param name="url">
 			<xsl:choose>
 				<xsl:when test="string-length($section) != 0 and string-length($entry-id) != 0">
+					<xsl:value-of select="$domain" />
 					<xsl:text>/symphony/publish/</xsl:text>
 					<xsl:value-of select="$section"/>
 					<xsl:text>/edit/</xsl:text>
 					<xsl:value-of select="$entry-id" />
 					<xsl:text>/</xsl:text>
 				</xsl:when>
 				<xsl:otherwise>
-					<xsl:value-of select="concat($root, '/symphony/')" />
+					<xsl:choose>
+						<xsl:when test="string-length($domain) != 0">
+							<xsl:value-of select="$domain" />
+						</xsl:when>
+						<xsl:otherwise>
+							<xsl:value-of select="$root"/>
```

Comment:
Why put `$root` in once place but not in the other ? I would vote for $root always being included, since it will always work (even with a custom domain)...

So I would put the default value of domain to be `$root` and remove the choose.

I would also rename the parameter to be "prefix" or "url-prefix", since it can contains more than just the domain.

Reply:
Fixed